cypress/e2e/app/simple-e2etest.cy.js

```diff
@@ -70,44 +72,50 @@ describe("End to end test using a simple UI path through the app", () => {
                 // B. Categorize "participant_id" as "Subject ID"
                 cy.categorizeColumn("Subject ID", p_dataset["category_columns"]["Subject ID"][0]);
 
-                // C. Assert nav and next button are enabled
-                cy.assertNextPageAccess("annotation", true);
+                // C. Assert nav and next button are not yet enabled
```

Comment:
Could you use this comment to explain **why** they shouldn't be enabled yet? I can read the code in the next line to understand that this is **what** is being asserted, but I don't know why I would expect that assertion to be true. 

I'm assuming it is because the app doesn't consider just labeling participant and/or session as sufficient for an annotation, i.e. they are required, but not sufficient.

Reply:
Yea this is due to the changes made to `isPageAccessible` getter. I'll add a comment.